spec/features/transactions_filter_spec.rb

```diff
@@ -27,7 +27,7 @@
 
     before do
       visit root_path
-      click_on 'Filters'
+      find(:css, 'a#toggle_filter').trigger('click')
```

Comment:
@yunixon it should work with `js: true` too. I used it many times in other projects